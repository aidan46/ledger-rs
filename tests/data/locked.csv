type,       client, tx, amount
deposit,         2,  2,    2
withdrawal,      2,  3,    1.5
dispute,         2,  2,
chargeback,      2,  2,
