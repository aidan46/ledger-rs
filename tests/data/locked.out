client,	available, held, total, locked
2,	-1.5,	   0,    -1.5,	true
